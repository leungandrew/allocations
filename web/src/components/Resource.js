import React from 'react';

const Resource = ({resource}) => {
  return (
    <div>
      {resource.firstName} {resource.lastName}
    </div>
  )
};

export default Resource;
