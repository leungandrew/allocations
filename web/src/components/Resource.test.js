import React from 'react';
import Resource from './Resource';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

let resource;

describe('Resource', () => {

  beforeEach(() => {
    resource = {
        id: 'id-1',
        firstName: 'FirstUser1',
        lastName: 'LastUser1'
      }
  });

  it('renders a Resource', () => {
    const subject = shallow(<Resource resource={resource} />);
    expect(toJson(subject)).toMatchSnapshot();
  });
});

