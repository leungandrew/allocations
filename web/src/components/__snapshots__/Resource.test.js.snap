// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resource renders a Resource 1`] = `
<div>
  FirstUser1
   
  LastUser1
</div>
`;
